app/client/liveChat.ts

```diff
@@ -0,0 +1,68 @@
+// <style type='text/css'>
+//     .embeddedServiceHelpButton .helpButton .uiButton {
+//         background-color: #005290;
+//         font-family: "Arial", sans-serif;
+//     }
+//     .embeddedServiceHelpButton .helpButton .uiButton:focus {
+//         outline: 1px solid #005290;
+//     }
+// </style>
+
+// tslint:disable:variable-name
+// tslint:disable:no-object-mutation
+const initESW = (gslbBaseUrl: string | null, embedded_svc: any) => {
+  embedded_svc.settings.displayHelpButton = true; // Or false
+  embedded_svc.settings.language = ""; // For example, enter 'en' or 'en-US'
+
+  // embedded_svc.settings.defaultMinimizedText = '...'; //(Defaults to Chat with an Expert)
+  // embedded_svc.settings.disabledMinimizedText = '...'; //(Defaults to Agent Offline)
+
+  // embedded_svc.settings.loadingText = ''; //(Defaults to Loading)
+  // embedded_svc.settings.storageDomain = 'yourdomain.com'; //(Sets the domain for your deployment so that visitors can navigate subdomains during a chat session)
+
+  // Settings for Chat
+  // embedded_svc.settings.directToButtonRouting = function(prechatFormData) {
+  // Dynamically changes the button ID based on what the visitor enters in the pre-chat form.
+  // Returns a valid button ID.
+  // };
+  // embedded_svc.settings.prepopulatedPrechatFields = {}; //Sets the auto-population of pre-chat form fields
+  // embedded_svc.settings.fallbackRouting = []; //An array of button IDs, user IDs, or userId_buttonId
+  // embedded_svc.settings.offlineSupportMinimizedText = '...'; //(Defaults to Contact Us)
```

Comment:
This is part of the generated script. For now it will be left in but will be updated in due course

Reply:
We do the same in other places. That is fine. Alternatively you can turn the rule back on after you've done what you needed to do. But best test if that works correctly as I remember tslint being weird on those cases. This will ensure we can't miss out the rule for the rest of the file (which is fine now but that might change when someone next touches it)